src/explore-education-statistics-admin/src/pages/users/ManageUserPage.tsx

```diff
@@ -111,14 +118,90 @@ const ManageUserPage = ({
                   />
                 </FormFieldset>
 
-                {/* <FormFieldCheckboxGroup
-                id={`${formId}-releaseaccess`}
-                legend="Release access"
-                legendSize="m"
-                hint="The releases a user has access to (BAU Users have access to all releases)."
-                name="releaseAccess"
-                options={[]}
-              /> */}
+                <FormFieldset
+                  id={`${formId}-role`}
+                  legend="Release access"
+                  legendSize="m"
+                  hint="The releases a user can access within the service."
+                >
+                  {/* <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-one-half">
+                      <FormFieldSelect
+                        id={`${formId}-selectedReleaseId`}
+                        label="Release"
+                        name="selectedReleaseId"
+                        options={releases?.map(release => ({
+                          label: `${release.publicationTitle} - ${release.releaseName}`,
+                          value: release.id,
+                        }))}
+                      />
+                    </div>
+
+                    <div className="govuk-grid-column-one-quarter">
+                      <FormFieldSelect
+                        id={`${formId}-releaseRoleId`}
+                        label="Release role"
+                        name="releaseRoleId"
+                        options={releaseRoles?.map(releaseRole => ({
+                          label: releaseRole.name,
+                          value: releaseRole.value,
+                        }))}
+                      />
+                    </div>
+                    <div className="govuk-grid-column-one-quarter">
+                      <Button type="submit" className="govuk-!-margin-top-6">
+                        Add release access
+                      </Button>
+                    </div>
+                  </div> */}
+
+                  <table className="govuk-table">
+                    <thead className="govuk-table__head">
```

Comment:
You don't really need to include any of the `govuk-table__` classes as styling is already applied by the parent table's `govuk-table` class.